### DISCUSSION TITLE
Using in conjunction with shadcn/ui

### DISCUSSION THREAD

**Discussion by avin-kavish** [heroui-inc/heroui#1367] [Q&A]
Hello, I've been using v1 with shadcn. Need to upgrade to v2 as v1 doesn't work with RSC. 

It looks like v2 styles might clash with shadcn. In your docs I noticed there are classes like,

```
<main className="dark text-foreground bg-background">
```

which are identical to shadcn. So I'm wondering if I can add a prefix or something to these classes so there's no conflicts.

**Comment by jrgarciadev** [comment]
We currently don't provide a way to prefix the semantic classes it would lead you to a bigger css bundle, maybe you could add the same semantic colors (foreground/background) to shadcn and nextui

**Comment by avin-kavish** [comment]
Right, actually I'm using slightly different shades of color in different locations. But since shadcn is copy and paste code, I've edited the classes on that side.
